src/main/java/com/techsophy/tsf/wrapperservice/service/impl/ProcessServiceImpl.java

```diff
@@ -413,6 +415,62 @@ public void completeTaskWithChecklistItemId(ChecklistItemInstanceDTO checklistIt
 
     }
 
+    @Override
+    public void deleteProcessById(String ticketNumber,String id) throws JsonProcessingException
+    {
+        String processInstanceId="";
+        Map<String,String> ticketData =new HashMap<>();
+        String ticketNumberValue =REVIEW_TICKET+ticketNumber;
```

Comment:
why is this concatenation required?

Reply:
If the raised ticket at review state then only End user has a permission to cancel that particular ticket .